addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/__tests__/environment-sc-connection-service.test.js

```diff
@@ -503,6 +503,48 @@ describe('EnvironmentScConnectionService', () => {
       expect(retConn).toBe(connection);
     });
 
+    it('should return ssm connection', async () => {
+      // BUILD
+      const connection = { info: 'This is ssm connection' };
+      service.mustFindConnection = jest.fn(() => connection);
+
+      envScService.mustFind = jest.fn(() => {
+        return {
+          outputs: {
+            WorkspaceSSMRoleArn: 'WorkspaceSSMRoleArn',
+            Ec2WorkspaceInstanceId: 'WorkspEc2WorkspaceInstanceIdaceSSMRoleArn',
+            SessionDuration: 3599,
+          },
+        };
+      });
+      const aws = await service.service('aws');
+      aws.getCredentialsForRole = jest.fn(() => {
+        return {
+          accessKeyId: 'accessKeyId',
+          secretAccessKey: 'secretAccessKey',
+          sessionToken: 'sessionToken',
+        };
+      });
+
+      service.generateSigninSSMConsoleLink = jest.fn(
+        () =>
+          'https://us-east-1.signin.aws.amazon.com/federation?Action=login&Issuer=Instruqt&Destination=https://us-east-1.console.aws.amazon.com/systems-manager/session-manager/i-xxx?region=us-east-1&SigninToken=xxx',
```

Comment:
Mocking the to be tested method with values does not make sense.
You should mock the aws sdk call then testing the other logics in that method.

See https://aws.amazon.com/blogs/developer/mocking-modular-aws-sdk-for-javascript-v3-in-unit-tests/ for mocking aws sdk api.